types/react-loader/index.d.ts

```diff
@@ -0,0 +1,45 @@
+// Type definitions for react-loader 2.4
+// Project: https://github.com/quickleft/react-loader
+// Definitions by: Sudarsan Balaji <https://github.com/artfuldev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'react-loader' {
```

Comment:
Do not use declare module, just remove this and the lint rule exception and things should work.